operator/EXTENDING_CRDS.md

```diff
@@ -101,3 +101,25 @@ Some data types are discouraged, in particular:
 * Nullability: See [Kubernetes Documentation](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#defaulting-and-nullable).
 * Late initialization: See [API Conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#late-initialization).
 * Labels, Selector and Annotations: See [API conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#label-selector-and-annotation-conventions).
+
+## Breaking changes
```

Comment:
Perhaps this should even go before the "Style recommendations" section

Reply:
Agree, done